src/components/atoms/List/List.tsx

```diff
@@ -4,15 +4,14 @@ import ListItem from 'components/atoms/ListItem';
 import { StyledList } from './List.styled';
 import { IListElement } from './List.model';
 
-export interface IProps {
+export interface IProps extends React.HTMLProps<HTMLUListElement> {
   icon: string;
   items: IListElement[];
-  className?: string;
 }
 
-const List = ({ icon, items, className }: IProps): JSX.Element => {
+const List = ({ icon, items }: IProps): JSX.Element => {
```

Comment:
Make sure that it's not interrupt styling for list when we remove className. Maybe we should have possibility to add rest props there

```suggestion
const List = ({ icon, items, ...rest }: IProps): JSX.Element => {
	return (
		<StyledList data-testid="list" {...rest}>
```